plugins/aglight/export.js

```diff
@@ -0,0 +1,168 @@
+/* eslint-disable radix */
+/* Based on the ofl export plugin */
+
+const fixtureJsonStringify = require(`../../lib/fixture-json-stringify.js`);
+const colors = require(`./colors`);
```

Comment:
Please include the file extension.

```suggestion
const colors = require(`./colors.js`);
```